languages/r34/template/src/Algorithm.r

```diff
@@ -0,0 +1,15 @@
+library(algorithmia)
+# API calls will begin at the algorithm() method, with the request body passed as 'input'
+# For more details, see algorithmia.com/developers/algorithm-development/languages
+
+algorithm <- function(input) {
+  paste("hello", input)
+}
+
+load <- function() {
+  # Defines the loading function if defined
+}
+
+algo <- getAlgorithmHandler(algorithm, load)
+algo$run()
```

Comment:
Didn't you use `serve` instead of `run` in the final Java implementation (or am I mistaken)?

Reply:
oh yes good catch will rename